packages/amplify-e2e-core/src/init/initProjectHelper.ts

```diff
@@ -149,6 +150,41 @@ export function initIosProjectWithProfile(cwd: string, settings: Object) {
   });
 }
 
+export function initFlutterProjectWithProfile(cwd: string, settings: Object) {
+  const s = { ...defaultSettings, ...settings };
+
+  return new Promise((resolve, reject) => {
+    let chain = spawn(getCLIPath(), ['init'], { cwd, stripColors: true })
+      .wait('Enter a name for the project')
+      .sendLine(s.name)
+      .wait('Enter a name for the environment')
+      .sendLine(s.envName)
+      .wait('Choose your default editor:')
+      .sendLine(s.editor)
+      .wait("Choose the type of app that you're building")
+      .send('j')
+      .send('j')
+      .sendCarriageReturn()
+      .wait('Where do you want to store your configuration file')
+      .sendLine('./lib/')
+      .wait('Using default provider  awscloudformation')
+      .wait('Do you want to use an AWS profile?')
+      .sendLine('y')
```

Comment:
We added the `confirmYes` and `confirmNo` please use those.

Reply:
Done.